### PR TITLE
feat: Upload v3

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**

### CODE REVIEW

File: kDriveCore/Data/Cache/DriveFileManager.swift [hunk 1]
```diff
@@ -49,7 +49,7 @@ public final class DriveFileManager {
         static let upload: UInt64 = 18
 
         /// Current version of the Drive Realm
-        static let drive: UInt64 = 9
+        static let drive: UInt64 = 10
```
Comment (PhilippeWeidmann) [inline kDriveCore/Data/Cache/DriveFileManager.swift:52]: This seems wrong, I think you meant `static let upload: UInt64 = 19` ?

Comment (adrien-coye) [reply kDriveCore/Data/Cache/DriveFileManager.swift:52]: Good catch, I tend to mix the two. Fixed in next commit.

**Comment by PhilippeWeidmann** [comment]
LGTM. Waiting for other branches to merge 👍

**Review by adrien-coye — DISMISSED** [review]
all good. 🚀

**Pull request merged**